app/presenters/users/dashboard_presenter.rb

```diff
@@ -49,36 +49,31 @@ def courses
             )
           else
             current_school.courses.live.where(
-              id:
-                (
-                  courses_with_student_profile.pluck(:course_id) +
-                    courses_with_review_access + courses_with_author_access
-                ).uniq
+              id: (courses_with_student_profile.pluck(:course_id) +
+                   courses_with_review_access + courses_with_author_access).uniq
```

Comment:
Incorrect formatting, please use the [recommended config](https://docs.pupilfirst.com/developers/development_setup#code-formatting)